esp32-c3-lcdkit/idf_component.yml

```diff
@@ -0,0 +1,25 @@
+version: "1.0.0"
+description: Board Support Package for esp32-c3-lcdkit
+url: https://github.com/espressif/esp-bsp/tree/master/esp32-c3-lcdkit
+
+targets:
+  - esp32c3
+
+dependencies:
+  idf:
+    version: ">=5.0.0"
```

Comment:
Must support ESP-IDF: 4.4 here?
I'm not sure yet if there will be some compatibility issues.

Reply:
We decided, that we should support IDF4.4 and higher in future in BSP. The first BSP is LyraT: https://github.com/espressif/esp-bsp/tree/master/esp32_lyrat you can see, that there are two new files, which solving compatibility issues (mainly in audio I2S). We will do it for all BSPs in next months. 
If you don't support it now, we will have to do it in future.

Reply:
> We decided, that we should support IDF4.4 and higher in future in BSP. The first BSP is LyraT: https://github.com/espressif/esp-bsp/tree/master/esp32_lyrat you can see, that there are two new files, which solving compatibility issues (mainly in audio I2S). We will do it for all BSPs in next months. If you don't support it now, we will have to do it in future.

LGTM，I think I can‘t finish it today, I will add it next week.

Reply:
Ok, thank you. But we have bank holiday next week from wednesday. If you need the BSP soon, we can merge it today (or monday) and make new PR for changes for IDF 4.4. Do you want to merge it today or wait for all changes?

Reply:
> Ok, thank you. But we have bank holiday next week from wednesday. If you need the BSP soon, we can merge it today (or monday) and make new PR for changes for IDF 4.4. Do you want to merge it today or wait for all changes?

It is definitely better if it can be released first. so that I can update my project files.
4.4 support is not very urgent for me now.

Reply:
ok, I will merge it today. Please, don't forget update this BSP with IDF4.4 support in future. Please create Jira task for it and put the number here for reference. Thank you